CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.0
+ - new feature: Add new option "push_map_as_event_on_timeout" so that when events time out they can be yielded as new events
```

Comment:
I think this description could be better : "so that when task timeout happens, the aggregated map can be yielded as a new event".
What do you think ?